modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/user/application/v1/UserDiscoverableApplicationSuccessTest.java

```diff
@@ -43,6 +42,8 @@ public class UserDiscoverableApplicationSuccessTest extends UserDiscoverableAppl
 
     private static final String PAGINATION_LINK_QUERY_PARAM_STRING = "?offset=%d&limit=%d";
     private static final String APP_NAME_WITH_SPACES = "APP_SPACES IN NAME";
+    // Store application with spaces in the name.
+    ApplicationResponseModel application;
```

Comment:
let's make this private

Reply:
addressed by https://github.com/wso2/product-is/pull/23086/commits/89fe2664f35ecdd47e52e91b435d915a170577e5